content/en/docs/troubleshooting/_index.md

```diff
@@ -35,6 +35,39 @@ Failed to create pod sandbox: rpc error: code = Unknown desc = failed to create
 Unfortunately, because this is a generic message, you'll need to go deeper to figure out
 what is going on.
 
+### Debug container
+
+It can be quite usefull to start a debug container, usually kubernetes nodes are stripped / hardened, which means no tools are installed, a debug container can be any image.
+
+`Debug container`
+```sh
+kubectl debug node/dev01-worker -it --image=busybox
+```
+
+When in a debug container host system are mounted under */host*
+
+`Edit containerd config`
+```sh
+vi /host/etc/containerd/config.toml
+```
+
+`Show containerd log`
+```sh
+chroot /host
+journalctl -xeu containerd
+```
+
+`Edit kata config`
+```sh
+vi /host/opt/kata/containerd/config.d/kata-deploy.toml
```

Comment:
I think after calling `chroot /host` one can't access the files with the `/host` prefix. 
So may be the `chroot /host` statement can be moved immediately after running the `kubectl debug` command (Line: 44) and then referring to all the config filess without the `/host` prefix.

Reply:
I had missed this in the earlier PR. Sorry about it..

Reply:
Hmm I see.. 
All the edit operations post running of `chroot /host` will fail. So you might want to add a comment clarifying this